packages/shared/src/hooks/useCreateSquadModal.ts

```diff
@@ -2,16 +2,21 @@ import { useRouter } from 'next/router';
 import { useEffect, useMemo, useRef } from 'react';
 import { LazyModal } from '../components/modals/common/types';
 import { useLazyModal } from './useLazyModal';
+import usePersistentContext from './usePersistentContext';
 
 interface UseCreateSquadModal {
   openNewSquadModal: () => void;
   openSquadBetaModal: () => void;
 }
 
+const SQUAD_ONBOARDING = 'hasTriedSquadOnboarding';
```

Comment:
Can we keep all the persistent context flags in somewhere common (maybe with an enum)?

Reply:
It's a good idea, I'll lodge a ticket for it.
Generally now they are in the close files to avoid imports, but think it's starting to make sense.